Add portfolio pages (#96)
octopusthink/octopusthink.com: src/content/portfolio/nautilus.mdx

  alt="User interview script and a screen grab from a session recording"
  next="spacing"
  >

  Nautilus includes a flexible type system, including rules for calculating type scales and for programmatically determining an optimal line-height that would still fall on our baseline grid.

  Semantic combinations for different "voices" and contexts forms the baseline of the type system. Type variables are themeable, as are the actual combinations of tokens into semantic styles, making for a highly flexible system with core rules.
</PortfolioSection>

</PortfolioSection>
<PortfolioSection
  metadata="Grids, gutters, and rhythm"
  heading="Spacing"
  image="/portfolio/nautilus/spacing.png"
  alt="Screenshots of a database of research insights"
  next="react-components"
  >

  Nautilus uses the popular `8px` grid due to its flexibility and ease of use with icons and other elements. It also includes spacing variables and rules for how padding and margin is applied to elements. This leads to more harmonious visual layouts.
</PortfolioSection>

</PortfolioSection>
<PortfolioSection
  metadata="Now for the easy part"
  heading="React components"
  image="/portfolio/nautilus/components.png"

>   alt="Screenshots from user research report, highlighting user journey map."

Reviewer: I won't leave comments on every single one, but please run through the `alt` text in your portfolio pieces and make sure they're right and not copypasta 😉

---
id: nautilus
title: Nautilus Design System
order: 1
thumbnail: '/portfolio/nautilus/styleguide-screenshot.png'
background: '/portfolio/nautilus/background-pattern.png'
tags: [documentation, leadership, design-systems, visual-design, React, project management, accessibility, design-principles]
summary: An open-source design system and React component library to help designers and developers build better digital products together. Designed for inclusivity, usability, and flexibility.
repo: https://nautilus.octopusthink.com/
---
import ImagePanel from 'components/ImagePanel';
import PortfolioSection from 'components/PortfolioSection';

<PortfolioSection
  metadata="Introduction"
  heading="The challenge"
  image="/portfolio/nautilus/venn-diagram.png"
  alt="Venn diagram showing: What designers know (typography), what developers know (React), and the intersection (shared vocabulary)."
  >

>   I can't count the number of times I've designed and built a button over the years. And to what end? They usually wind up looking (and acting!) more or less the same. As someone who likes to build things and work on side projects, I wanted a way to be able to quickly get up and running with a new idea.

Reviewer: I think the use of "I" in here could be appropriate, but if it stays this needs to be attributed. Otherwise it sounds out-of-place on a website for the business. I know this was chiefly a "design" problem, though I felt similarly with regards to "building" the button. This either needs to be:

* third-person, eg: _Sarah couldn't count the number of times she'd designed and built a button over the years._
* attributed somehow

>   I also need to have a shared vocabulary and set of practises to work better with my developer partner-in-crime. Codifying and documenting our practises allows us both to build things independently.

Reviewer: "developer partner-in-crime" could probably just be "developers" or "more technical people", etc.

>   Creating a design system allows me to establish a solid, predictable baseline for my work without needing to reinvent the wheel every time. Instead of building yet-another-button, I can spend more of my time innovating on top of my existing baseline.

Reviewer: ```suggestion
  Creating a design system allows us to establish a solid, predictable baseline for our work without needing to reinvent the wheel every time. Instead of building yet-another-button, we can spend more of my time innovating on top of our already-excellent baseline.
```

</PortfolioSection>

<ImagePanel dark alt="A mockup showing screenshots of the design system documentation site on a desktop, tablet, and phone." image="/portfolio/nautilus/device-mockup-pdf.png" />

<PortfolioSection
  metadata="Establishing vision"
  heading="Yet another design system?"
  image="/portfolio/nautilus/structure-diagram.png"
  alt="Diagram of Nautilus' intended structure"
  >

>   There are loads of other, better-established design systems out there I could leverage. Why bother to build my own?

Reviewer: The use of "I" and "my" in here, again, doesn't work well for a collective portfolio. Given we built Nautilus together it's a bit off too. Probably best making these parts a bit more collective/"we"-based.

Author: Done with the copyediting!
